clang/test/Driver/apple-os-triples.c

```diff
@@ -0,0 +1,31 @@
+// Test triple manipulations.
+
+// RUN: %clang -### -c %s \
+// RUN:     -target i386-apple-darwin10 -mappletvsimulator-version-min=9.0 -arch x86_64 2>&1 | \
```

Comment:
Perhaps 2-space indentation, which is much more common.

(`-target ` has been deprecated for a long time at least for ELF. I guess Apple OS might want to have an exception since they also have `-arch `)